### PR TITLE
Add swagger section

### PR DESCRIPTION
**Pull request by darkBasic666**
## Summary
Add API documentation section

### CODE REVIEW

File: backend/backend-standard-and-best-practices.md [hunk 1]
```diff
@@ -213,7 +214,19 @@ Most commonly used are:
 
 You can read more about this and other status codes in [this link](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status).
 
-### 6- Routes
+## 6- API Documentation
+
+There are lots of different tools in the market to document APIs. Here in Wolox we decided to use Openpi 3 (A.K.A Swagger) as a standard documentation tool in all of our techs. The reasons behind this decision were some of the advantages OpenApi prodvides:
```
Comment (icoluccio) [inline backend/backend-standard-and-best-practices.md]: Openapi*

File: backend/backend-standard-and-best-practices.md [hunk 2]
```diff
@@ -213,7 +214,19 @@ Most commonly used are:
...
+
+* **Quick to use**: The properties that you need to complete in every endpoint are always the same and there are only a few. You can also frequently reuse components and the generic information is simple to complete.
+* **Wide range of available features**: From describing parameters, making authentication and authorization schemes, to even using the generated documentation of an endpoint to mock test the endpoint in question.
+* **Intuitive and easy to use interactive UI**: Swagger up a view where show how to do a request and how to response the API. 
+* **Maintainable and easily extendable**: Swagger permit use constants file/s from your API.
```
Comment (icoluccio) [inline backend/backend-standard-and-best-practices.md]: allows using?

File: backend/backend-standard-and-best-practices.md [hunk 3]
```diff
@@ -213,7 +214,19 @@ Most commonly used are:
...
 You can read more about this and other status codes in [this link](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status).
 
-### 6- Routes
+## 6- API Documentation
+
+There are lots of different tools in the market to document APIs. Here in Wolox we decided to use Openpi 3 (A.K.A Swagger) as a standard documentation tool in all of our techs. The reasons behind this decision were some of the advantages OpenApi prodvides:
+
+* **Quick to use**: The properties that you need to complete in every endpoint are always the same and there are only a few. You can also frequently reuse components and the generic information is simple to complete.
+* **Wide range of available features**: From describing parameters, making authentication and authorization schemes, to even using the generated documentation of an endpoint to mock test the endpoint in question.
+* **Intuitive and easy to use interactive UI**: Swagger up a view where show how to do a request and how to response the API.
```
Comment (icoluccio) [inline backend/backend-standard-and-best-practices.md]: I think you missed a few words here

**Review by icoluccio — CHANGES_REQUESTED** [review]

**Review by KevinTemes — APPROVED** [review]

File: backend/backend-standard-and-best-practices.md [hunk 4]
```diff
@@ -213,7 +214,19 @@ Most commonly used are:
 
 You can read more about this and other status codes in [this link](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status).
 
-### 6- Routes
+## 6- API Documentation
+
+There are lots of different tools in the market to document APIs. Here in Wolox we decided to use Openapi 3 (A.K.A Swagger) as a standard documentation tool in all of our techs. The reasons behind this decision were some of the advantages OpenApi prodvides:
+
+* **Quick to use**: The properties that you need to complete in every endpoint are always the same and there are only a few. You can also frequently reuse components and the generic information is simple to complete.
```
Comment (zeminlu) [inline backend/backend-standard-and-best-practices.md]: you need to complete -> you  need to fill
simple to complete -> simple to provide

File: backend/backend-standard-and-best-practices.md [hunk 5]
```diff
@@ -213,7 +214,19 @@ Most commonly used are:
...
+* **Wide range of available features**: From describing parameters, making authentication and authorization schemes, to even using the generated documentation of an endpoint to mock test the endpoint in question.
+* **Intuitive and easy to use interactive UI**: Swagger uses a GUI to show the user how to make requests to the API and what responses will be like.
```
Comment (zeminlu) [inline backend/backend-standard-and-best-practices.md]: will be like -> will look like

File: backend/backend-standard-and-best-practices.md [hunk 6]
```diff
@@ -213,7 +214,19 @@ Most commonly used are:
...
 
-### 6- Routes
+## 6- API Documentation
+
+There are lots of different tools in the market to document APIs. Here in Wolox we decided to use Openapi 3 (A.K.A Swagger) as a standard documentation tool in all of our techs. The reasons behind this decision were some of the advantages OpenApi prodvides:
+
+* **Quick to use**: The properties that you need to complete in every endpoint are always the same and there are only a few. You can also frequently reuse components and the generic information is simple to complete.
+* **Wide range of available features**: From describing parameters, making authentication and authorization schemes, to even using the generated documentation of an endpoint to mock test the endpoint in question.
+* **Intuitive and easy to use interactive UI**: Swagger uses a GUI to show the user how to make requests to the API and what responses will be like. 
+* **Maintainable and easily extendable**: Swagger allows using constants file/s from your API.
```
Comment (zeminlu) [inline backend/backend-standard-and-best-practices.md]: file/s -> files

**Review by zeminlu — CHANGES_REQUESTED** [review]
Just a couple of changes and all set!

**Review by zeminlu — APPROVED** [review]

**Review by icoluccio — APPROVED** [review]

**Pull request merged**